apps/meteor/app/apps/server/converters/rooms.js

```diff
@@ -337,6 +379,54 @@ export class AppRoomsConverter {
 		return transformMappedData(originalRoom, map);
 	}
 
+	convertRoomWithoutLookups(room) {
+		const creatorFromRoom = room.u
+			? {
+					_id: room.u._id,
+					username: room.u.username,
+					name: room.u.name || room.u.username || 'Unknown',
+				}
+			: undefined;
+
+		const fallbackUser = {
+			_id: 'unknown',
+			username: 'unknown',
+			name: 'Unknown',
+		};
```

Comment:
Since this method is converting an existing room to match internal apps-engine types, business rules guarantee the `u` property is present, no need to walk around it like this or have a fallback - this fallback is really not helpful at all, it would be better to send `undefined` to the app anyway. But since we'll always have a `u`, there is no need for it

Reply:
I forgot to remove the unused method `convertRoomWithoutLookups`.